### ISSUE TITLE
Motif-Centric Representation Learning for Symbolic Music

### ISSUE THREAD

**Issue by InabaTatsuro** [InabaTatsuro/papers#66]
# Motif-Centric Representation Learning for Symbolic Music [arXiv23/9]

<img src="https://github.com/InabaTatsuro/papers/assets/102784221/0cfa6732-bbb7-41f0-ba63-78f60874362b" width="400">

## Abst
- Use VICReg, the training method in which no negative samples are used, for pretraining the model and use contrastive learning for finetuning
- to reduce the influence of randomly selected negative samples that have similar segments with positive samples

## VICReg[[ICLR22](https://arxiv.org/abs/2105.04906)]
<img src="https://github.com/InabaTatsuro/papers/assets/102784221/0476728f-59fc-4237-bd79-8fb3bb8ca700" width="400">

- Invariance loss: pushing positive pairs closer

<img src="https://github.com/InabaTatsuro/papers/assets/102784221/58f2cdd3-a36b-456d-aba8-40edc3263a21" width="400">

- Variance loss: maintain sufficient variance within each batch

<img src="https://github.com/InabaTatsuro/papers/assets/102784221/3cbe9966-874e-49f2-a7c2-e8fe95ca91a0" width="400">

- force each dimension of embeddings to have distinct information

## Architecture
<img src="https://github.com/InabaTatsuro/papers/assets/102784221/97962b94-861f-4a6b-a0f4-72c1d38cffb7" width="400">

- expander is eliminated after training
